contrib/automation_tests/test_cases/symbols_tab.py

```diff
@@ -53,15 +266,22 @@ def _execute(self, module_search_string: str):
         logging.info('Waiting for * to indicate loaded modules')
 
         # The Loading column which should get filled with the "*" is the first column (0)
-        wait_for_condition(lambda: modules_dataview.get_item_at(0, 0).texts()[0] == "*", 100)
+        start_time = time.time()
+        if expect_fail:
+            wait_for_condition(lambda: _find_and_close_error_dialog(self.suite.top_window()) is not None)
+        else:
+            wait_for_condition(lambda: modules_dataview.get_item_at(0, 0).texts()[0] == "*", 100)
+        total_time = time.time() - start_time
+        self._check_and_update_duration("load_symbols_" + module_search_string, total_time,
+                                        expected_duration_difference)
 
-        functions_dataview = DataViewPanel(self.find_control("Group", "FunctionsDataView"))
-        wait_for_condition(lambda: functions_dataview.get_row_count() > 0)
+        VerifySymbolsLoaded(symbol_search_string=module_search_string, expect_loaded=not expect_fail).execute(
+            self.suite)
 
 
 class VerifyModuleLoaded(E2ETestCase):
     """
-    Verifies wether a module with the give search string is loaded.
+    Verifies whether a module with the give search string is loaded.
```

Comment:
Lol, thanks for fixing this. I send feedback to google translate [where I checked  this](https://www.google.com/search?q=wether). Let's see if they fix it upstream.

Reply:
"A castrated ram" TIL... xD